packages/@react-spectrum/menu/src/Menu.tsx

```diff
@@ -96,6 +98,7 @@ function Menu<T extends object>(props: SpectrumMenuProps<T>, ref: DOMRef<HTMLULi
           <ul
             {...menuProps}
             {...styleProps}
+            style={style}
```

Comment:
this is going to overwrite line 100 I assume based on the names

Reply:
Updated to merge these, thanks.